src/ucp/rndv/rndv_ats.c

```diff
@@ -34,18 +34,14 @@ static void ucp_proto_rndv_ats_probe(const ucp_proto_init_params_t *init_params)
         return;
     }
 
-    status = ucp_proto_rndv_ack_init(&params, UCP_PROTO_RNDV_ATS_NAME,
-                                     UCS_LINEAR_FUNC_ZERO, &perf, &priv);
-    if ((status != UCS_OK) || (priv.lane == UCP_NULL_LANE)) {
+    status = ucp_proto_rndv_ack_init(&params, UCP_PROTO_RNDV_ATS_NAME, 0, &perf,
+                                     &priv);
+    if ((status != UCS_OK) || (perf == NULL)) {
```

Comment:
can status be UCS_OK and perf==NULL? i would expect status to be an error in this case

Reply:
status can be OK and perf can be NULL in case if `UCP_PROTO_SELECT_OP_FLAG_PPLN_FRAG` is set (protocol trying to be initialized for using as a part of pipeline)